x-pack/plugins/cloud_security_posture/common/schemas/benchmark.ts

```diff
@@ -56,7 +56,7 @@ export const benchmarksQueryParamsSchema = schema.object({
   /**
    * Benchmark filter
    */
-  benchmark_name: schema.maybe(schema.string()),
+  package_policy_name: schema.maybe(schema.string()),
```

Comment:
any particular reason for this name change? it looks like the param name was changed but the files, dirs and types are still named benchmarks. i think maybe dont rename it in this pr if its not crucial

Reply:
Yes, while we used `benchmark_name` as input, the actual meaning was package policy name.
Now the input field name is aligned with what is actually support.

Reply:
if that's the case, should probably address the files, dirs and types as i mentioned